examples/redux-observable-shopping-cart/js/services/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Mocking client-server processing
+ */
+import _products from './products'
+
+const TIMEOUT = 100;
+const MAX_CHECKOUT = 2; // max different items
+
+export const api = {
+  getProducts() {
+    return new Promise( resolve => {
+      setTimeout(() => resolve(_products), TIMEOUT)
+    })
+  },
+
```

Comment:
It was intentional to leave it that way because the original example uses promises, so I tried to change only the middleware leaving the other code intact.
Maybe I could also add the original code so that a user could swap the epic with the saga and taste the difference.
The Observable since the beginning really rocks though... :-)  tell me what you think it's best I can change the code if you prefer.